platforms/Windows/rtl/lib/rtllib.wxs

```diff
@@ -38,6 +38,9 @@
       <Component>
         <File Source="$(SDK_ROOT)\usr\bin\swiftCRT.dll" />
       </Component>
+      <Component>
+        <File Source="$(SDK_ROOT)\usr\bin\swiftObservation.dll" />
```

Comment:
do we need to also provide the macros package too?

Reply:
I didn't know about the macros package!  Could you point me to that?  We will want to do that, but I think that for now it doesn't matter as macros aren't available yet on Windows (but working towards it).

Reply:
the macro is created from here: https://github.com/apple/swift/tree/main/lib/Macros/Sources/ObservationMacros